bookstore/client/src/app/_services/crud.service.ts

```diff
@@ -0,0 +1,37 @@
+import {Injectable} from '@angular/core';
+import {Http} from '@angular/http';
+import {Observable} from 'rxjs/Observable';
+
+@Injectable()
+export abstract class CrudService {
+  protected url = 'http://localhost:3000/';
+
+  protected model = '';
+
+  protected constructor(protected http: Http) {
```

Comment:
No is recommended used Http, is deprecated, the recommended is to use HttpClient